Class not found

```
[18:34:02 WARN]: java.lang.NoClassDefFoundError: org/apache/commons/lang3/tuple/ImmutablePair
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.module.modules.spawn.SpawnModuleBuilder.load(SpawnModuleBuilder.java:52)
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.module.ModuleFactory.build(ModuleFactory.java:165)
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.match.Match.registerModules(Match.java:52)
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.GameHandler.cycleAndMakeMatch(GameHandler.java:46)
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.GameHandler.<init>(GameHandler.java:33)
[18:34:02 WARN]: 	at in.twizmwaz.cardinal.Cardinal.onEnable(Cardinal.java:151)
[18:34:02 WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:315)
[18:34:02 WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:323)
[18:34:02 WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:416)
[18:34:02 WARN]: 	at org.bukkit.craftbukkit.v1_7_R4.CraftServer.loadPlugin(CraftServer.java:461)
[18:34:02 WARN]: 	at org.bukkit.craftbukkit.v1_7_R4.CraftServer.enablePlugins(CraftServer.java:395)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.MinecraftServer.n(MinecraftServer.java:362)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.MinecraftServer.g(MinecraftServer.java:336)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.MinecraftServer.a(MinecraftServer.java:292)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.DedicatedServer.init(DedicatedServer.java:189)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.MinecraftServer.run(MinecraftServer.java:446)
[18:34:02 WARN]: 	at net.minecraft.server.v1_7_R4.ThreadServerApplication.run(SourceFile:628)
[18:34:02 WARN]: Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.tuple.ImmutablePair
[18:34:02 WARN]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
[18:34:02 WARN]: 	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
[18:34:02 WARN]: 	at java.security.AccessController.doPrivileged(Native Method)
[18:34:02 WARN]: 	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
[18:34:02 WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:77)
[18:34:02 WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:62)
[18:34:02 WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
[18:34:02 WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
[18:34:02 WARN]: 	... 17 more
```
My Cardinal version is: ```This server is running CardinalPGM version 1.0-SNAPSHOT-git-CardinalPGM-a2659f9```

--- a/pom.xml
+++ b/pom.xml
@@ -54,6 +54,11 @@
             <artifactId>commons-io</artifactId>
             <version>2.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.0</version>
+        </dependency>
     </dependencies>
 
     <build>

--- a/pom.xml
+++ b/pom.xml
@@ -101,6 +101,7 @@
                                     <include>com.sk89q:command-framework-bukkit</include>
                                     <include>org.jdom:jdom2</include>
                                     <include>commons-io:commons-io</include>
+                                    <include>org.apache.commons:commons-lang3</include>
                                 </includes>
                             </artifactSet>
                         </configuration>

